examples/rps-7-rpc/client-py/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM       python:3.8-alpine
+FROM       {{PYTHON_IMAGE}}
```

Comment:
I don't like that this is a `mo` rather than an ARG... mainly because I don't like having two